From c83e75f548369e38e8f4c369717000552a8a1cdf Mon Sep 17 00:00:00 2001
From: James Youngman <jay@gnu.org>
Date: Sun, 3 Jan 2016 23:20:54 +0000
Subject: [PATCH 030/224] regexprops: Fix compiler warnings and update
 copyright years.

* lib/regexprops.c (newpara): Function parameter list should be
declared as void in the function definition, instead of leaving it
blank (C++ style).  This fixes a GCC warning controlled by
-Wstrict-prototypes.
(beginenum): Likewise.
(endenum): Likewise.
(endtable): Likewise.
(copying): Update copyright years.
* doc/regexprops.texi: Update copyright years to match the output
of regexprops.c.
---
 doc/regexprops.texi |  4 ++--
 lib/regexprops.c    | 12 ++++++------
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/doc/regexprops.texi b/doc/regexprops.texi
index 537b64e0..897a492d 100644
--- a/doc/regexprops.texi
+++ b/doc/regexprops.texi
@@ -1,5 +1,5 @@
-@c Copyright (C) 1994, 1996, 1998, 2000, 2001, 2003, 2004, 2005, 2006,
-@c 2007, 2009, 2010, 2011 Free Software Foundation, Inc.
+@c Copyright (C) 1994, 1996, 1998, 2000-2007, 2009-2011,
+@c 2015-2016 Free Software Foundation, Inc.
 @c
 @c Permission is granted to copy, distribute and/or modify this document
 @c under the terms of the GNU Free Documentation License, Version 1.3 or
diff --git a/lib/regexprops.c b/lib/regexprops.c
index 3409b4c4..2327a646 100644
--- a/lib/regexprops.c
+++ b/lib/regexprops.c
@@ -127,7 +127,7 @@ begintable_markup (char const *markup)
 }
 
 static void
-endtable ()
+endtable (void)
 {
   newline ();
   directive ("@end table");
@@ -135,7 +135,7 @@ endtable ()
 }
 
 static void
-beginenum ()
+beginenum (void)
 {
   newline ();
   directive ("@enumerate");
@@ -143,7 +143,7 @@ beginenum ()
 }
 
 static void
-endenum ()
+endenum (void)
 {
   newline ();
   directive ("@end enumerate");
@@ -151,7 +151,7 @@ endenum ()
 }
 
 static void
-newpara ()
+newpara (void)
 {
   content ("\n\n");
 }
@@ -453,8 +453,8 @@ copying (void)
 {
   static const char *copy_para[]=
     {
-      "Copyright (C) 1994, 1996, 1998, 2000, 2001, 2003, 2004, 2005, 2006,"
-      ,"2007, 2009, 2010, 2011 Free Software Foundation, Inc."
+      "Copyright (C) 1994, 1996, 1998, 2000-2007, 2009-2011,"
+      ,"2015-2016 Free Software Foundation, Inc."
       ,""
       ,"Permission is granted to copy, distribute and/or modify this document"
       ,"under the terms of the GNU Free Documentation License, Version 1.3 or"
-- 
2.19.1

